src/components/instantlaunches/index.js

```diff
@@ -57,6 +61,12 @@ export const InstantLaunchSubmissionDialog = ({ open }) => {
             </DialogTitle>
 
             <DialogContent>
+                {appInfo && (
+                    <Typography variant="body1">
+                        Launching app: {appInfo.name}
+                        {resource ? " with data " + resource.path : ""}
```

Comment:
These should be i18n messages, and it might be easiest to use 2 messages, something like this:
```suggestion
                        {resource
                            ? t("launchingAppWithData", {
                                  appName: appInfo.name,
                                  path: resource.path,
                              })
                            : t("launchingApp", { appName: appInfo.name })}
```

Reply:
Heh, yeah. I even commented yesterday that I should do this and then lost track of it. Pushing up a fix now, sorry about that